### ISSUE TITLE
Swiftlint plugin is triggering the plugin File Watcher

### ISSUE THREAD

**Issue by vegidio** [bealex/SwiftLintAppCode#51]
I recently installed the SwiftLint plugin in AppCode and I noticed that when I turn on the code inspection `Editor > Inspections > SwiftLint` then the watcher that I created on my File Watcher for `.swift` files is triggered automatically, as if the file is saved after every code inspection which is not the expected behavior

When SwiftLint inspection is off the File Watcher is not triggered.

This is my Inspections configuration:

![Inspections](https://user-images.githubusercontent.com/934172/101977231-53709400-3c4c-11eb-88ec-60984c45fd1f.png)

And this is my File Watcher configuration:

![File Watcher](https://user-images.githubusercontent.com/934172/101977224-4489e180-3c4c-11eb-93fd-200126af38f0.png)

**Comment by bealex** [comment]
Hi, thanks for the issue. Does this occur when you launch command line swiftlint for your project?

Plugin, basically, runs swiftlint on a current file when it is changed, and parses the result. Before running SwiftLint I have to dump caches to the file system, because swiftlint command works with a file itself.

I will see what I can do, but answer will help me to narrow down my options.

**Comment by vegidio** [comment]
Hi,

This is what I did to test:

1. I disabled the Swiftlint inspection on `Editor > Inspections > SwiftLint` and I also disabled the option `Save files when switching to a different application` (this is enabled by default on Appcode), otherwise Appcode would save the file automatically for me when I switch to the terminal do run `swiftlint` there.

<img width="881" alt="autosave" src="https://user-images.githubusercontent.com/934172/102097311-047e5680-3e26-11eb-897b-c53937bb1241.png">

2. I made sure that my file watcher is configured to be triggered by external changes, done outside Appcode (second screenshot in my first message).

3. I opened the terminal, I went to the project's folder and I ran `swiftlint`. I came back to Appcode and the File Watcher had not been triggered.

> Before running SwiftLint I have to dump caches to the file system, because swiftlint command works with a file itself.

One suggestion: maybe your plugin could save the changes in a temporary file in the file system and run `swiftlint` against this file? This way the original file will remain unaltered and the File Watcher will not be triggered.
